Makefile

```diff
@@ -57,12 +57,14 @@ all: stork storkctl cmdexecutor pretest px-statfs
 
 vendor-tidy:
 	go mod tidy
+	sed -i '1 i\// +build skipcompile\n' vendor/kubevirt.io/client-go/kubecli/kubevirt_test_utils.go
```

Comment:
why is this needed in "go mod tidy"? Should be enough to add it to "vendor"?

Reply:
Removed from `go mod tidy`